### PR TITLE
Add an indicator to say what your name is in the top right of the applicant view

### LINKED ISSUE #713: Better indicators for login / logout state.
**Describe the solution you'd like**
Display the logged in user's name in the upper right of the screen, and confirm that logging users out removes the functionality they had access to when logged in!

**Done when**
A browser test confirms that logging in shows your name (as grabbed from OIDC) and that logging out means you have no access to secure pages.

### PR DESCRIPTION
**Pull request by ndmckinley**
Screen Shot 2021-06-21 at 4 37 35 PM
### Description
This resolves #713.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)

### CODE REVIEW

File: universal-application-tool-0.0.1/app/services/MessageKey.java [hunk 1]
```diff
@@ -68,7 +68,8 @@
   TITLE_PROGRAM_REVIEW("title.programReview"),
   TOAST_APPLICATION_SAVED("toast.applicationSaved"),
   TOAST_LOCALE_NOT_SUPPORTED("toast.localeNotSupported"),
-  TOAST_PROGRAM_COMPLETED("toast.programCompleted");
+  TOAST_PROGRAM_COMPLETED("toast.programCompleted"),
+  NAME("header.userName");
```
Comment (carolinedanzi) [inline universal-application-tool-0.0.1/app/services/MessageKey.java:72]: super nit: `USER_NAME` to differentiate between other "name" things?

File: universal-application-tool-0.0.1/app/services/WellKnownPaths.java [hunk 2]
```diff
@@ -1,8 +1,8 @@
 package services;
 
 public class WellKnownPaths {
-  public static Path APPLICANT_FIRST_NAME = Path.create("applicant.name.first");
-  public static Path APPLICANT_MIDDLE_NAME = Path.create("applicant.name.middle");
-  public static Path APPLICANT_LAST_NAME = Path.create("applicant.name.last");
+  public static Path APPLICANT_FIRST_NAME = Path.create("applicant.name.first_name");
```
Comment (carolinedanzi) [inline universal-application-tool-0.0.1/app/services/WellKnownPaths.java:4]: question (non-blocking): Does changing this break anything? (previous submissions, etc.)

**Review by carolinedanzi — APPROVED** [review]
Nice work!

File: universal-application-tool-0.0.1/app/services/WellKnownPaths.java (hunk 2 above)
Comment (ndmckinley) [reply universal-application-tool-0.0.1/app/services/WellKnownPaths.java:4]: It is currently broken and this is fixing it!  This needs to stay in sync with the scalar locations in `NameQuestionDefinition`.

**Pull request merged**